### PR TITLE
Add Words Slide

### LINKED ISSUE #13: Add Words Slide
### Slide name

Words

### Slide information

`data.individual.words` has an array of words that each user uses commonly in their memos. Let's make a word cloud! 

Here's a good example of how to do this: https://github.com/hackclub/hcb-wrapped-2022/blob/6e9a8fe4a646acc264981f40a5e125aa74f7a7f2/wrapped.js#L127

### Code of Conduct

- [X] I agree to follow Hack Club's Code of Conduct

### PR DESCRIPTION
**Pull request by Aquoria**
Fixes #13

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
Great work on this @Aquoria; thank you so much!

For others' reference, here's a visual preview:

Screenshot 2023-12-14 at 2 24 31 AM

My only real comment is related to the header, could we:

* Center the text horizontally
* Make "words!" all caps and larger (about 20% larger font size)
* Reduce the vertical spacing between the lines (maybe about 12px?)

Otherwise, looking good to me. Thank you so much!

**Comment by Aquoria** [comment]
Thanks! I agree with the header stuff and it was originally centered, got thrown off for some reason but I thought it still looked fine. For consistency with the other slides I'll see if I can change that.

**Comment by Aquoria** [comment]
Updated CSS

SS 2023-12-14 at 6 54 53 AM

File: components/Wrapped/slides/WordCloud.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          textAlign: "center",
+          fontSize: 25
+        })}
+        id="input"
+      >
+        You used lots of
+      </h2>
+      <div
+        style={{
+          marginTop: -22,
```
Comment (sampoder) [inline components/Wrapped/slides/WordCloud.tsx:19]:
```suggestion
          marginTop: -22,
          marginBottom: 8
```

File: components/Wrapped/slides/WordCloud.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          marginBottom: 8
```
Comment (sampoder) [inline components/Wrapped/slides/WordCloud.tsx]:
```suggestion
          marginBottom: 8,
```

lol mb

**Review by sampoder — APPROVED** [review]
LGTM; incredible work on this one!

**Pull request merged**